pyhydroquebec/__main__.py

```diff
@@ -96,22 +100,42 @@ def main():
         print(VERSION)
         return 0
 
-    if not args.username or not args.password:
+    # Check input for Username, Password and Contract - CLI overwrite ENV variable
+    hydro_user = hydro_pass = hydro_contract = None
```

Comment:
If you use `os.environ.get...` things this line can be removed

Reply:
Done